apps/emqx_management/src/emqx_mgmt.erl

```diff
@@ -185,22 +186,33 @@ stopped_node_info(Node) ->
     {Node, #{node => Node, node_status => 'stopped', role => core}}.
 
 vm_stats() ->
-    Idle =
-        case cpu_sup:util([detailed]) of
-            %% Not support for Windows
-            {_, 0, 0, _} -> 0;
-            {_Num, _Use, IdleList, _} -> proplists:get_value(idle, IdleList, 0)
-        end,
-    RunQueue = erlang:statistics(run_queue),
+    Idle = vm_stats('cpu.idle'),
     {MemUsedRatio, MemTotal} = get_sys_memory(),
     [
-        {run_queue, RunQueue},
+        {run_queue, vm_stats('run.queue')},
```

Comment:
Curious: why don't we use the same atom format here, `'run.queue'`? We convert to it below.

Reply:
OpenTelemetry 's metric name  is format as `'x.y.z'` not `x_y_z`.
Prometheus's metrics name is format as `x_y_z`

Open telemetry Exporter will call vm_stats('run.queue') when collecting.
Prometheus will call vm_stats() when collecting.

So I make vm_stats/0 to call vm_stats/1